src/main/java/com/ust/sourcecourse/configuration/request/ColumnsRequest.java

```diff
@@ -0,0 +1,82 @@
+package com.ust.sourcecourse.configuration.request;
+
+public class ColumnsRequest {
+	private String name;
+	private String notes;
+	private String type;
+	private boolean isprimary;
+	private String prefix;
+	private String sufix;
+	private long dbid;
+	private String tableid;
+	private String Columnid;
```

Comment:
Only ColumnID is required. DB ID and TableID can be omitted from request. We would get these ids from Column ID anyway.